stellar-captive-core/debian/stellar-captive-core.defaults

```diff
@@ -0,0 +1,20 @@
+## /etc/default/stellar-horizon
+NETWORK_PASSPHRASE="Test SDF Network ; September 2015"
+DATABASE_URL="dbname=horizon user=stellar host=/var/run/postgresql"
+STELLAR_CORE_DATABASE_URL="dbname=stellar user=stellar host=/var/run/postgresql"
+STELLAR_CORE_URL="http://127.0.0.1:11626"
+
+# Captive Core Ingestion Config
+ENABLE_CAPTIVE_CORE_INGESTION=true
+STELLAR_CORE_BINARY_PATH=/usr/bin/stellar-core
+STELLAR_CORE_CONFIG_PATH=/etc/stellar/stellar-core.cfg
+# end Captive Core
+
+FRIENDBOT_SECRET=
+PORT=8000
+SENTRY_DSN=
+LOGGLY_TOKEN=
+PER_HOUR_RATE_LIMIT=9000
+INGEST=true
+HISTORY_ARCHIVE_URLS="https://history.stellar.org/prd/core-testnet/core_testnet_001"
+INTERNAL_PORT=4555
```

Comment:
I think most of the values here are not needed. I think the ones we need are listed [here](https://github.com/stellar/go/blob/master/exp/services/captivecore/README.md#usage). @tamirms to confirm.

Reply:
@bartekn I believe this configuration is *not* for the captive core api. The defaults for the API are in stellar-captive-core-api/debian/stellar-captive-core-api.default . This file defines the defaults for the captive-core debian package which is essentially horizon bundled with captive core as a subprocess. So all these configuration values should match the horizon configuration values